integration-tests/baas-test-server/cli.ts

```diff
@@ -95,12 +99,19 @@ yargs(hideBin(process.argv))
       docker.ensureNoBaas();
 
       if (argv.githash) {
-        docker.spawnBaaS({ tag: argv.githash, accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY });
+        docker.spawnBaaS({
+          image: argv.githash,
+          accessKeyId: AWS_ACCESS_KEY_ID,
+          secretAccessKey: AWS_SECRET_ACCESS_KEY,
+        });
+      } else if (argv["latest-local"]) {
+        const id = docker.getLatestLocalId();
+        docker.spawnBaaS({ image: id, accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY });
       } else {
         const tag = await docker.fetchBaasTag(argv.branch);
         assert(AWS_PROFILE, "Missing AWS_PROFILE env");
         docker.pullBaas({ profile: AWS_PROFILE, tag });
```

Comment:
We may want to rename this `tag` property to `image` if doing it for the `spawnBaaS()` argument.

Reply:
It's always a tag when it comes from baasaas 👍